src/main/java/jp/co/soramitsu/sora/crypto/common/DigestTypeEnum.java

```diff
@@ -0,0 +1,15 @@
+package jp.co.soramitsu.sora.crypto.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum DigestTypeEnum {
```

Comment:
add annotation @FieldDefaults, in this case you could to remove private and final keywords

Reply:
done